src/components/side_panel/chart/combo_chart/combo_chart_design_panel.xml

```diff
@@ -3,7 +3,16 @@
     <GeneralDesignEditor t-props="props">
       <t t-set-slot="general-extension">
         <ChartLegend t-props="props"/>
-        <ChartShowValues t-props="props"/>
+        <ChartShowValues t-props="props">
+          <t t-set-slot="values-extension">
```

Comment:
I get that we want to it next to "show value", but does it really make sense to have them in the same component ? As t-slot ? 

IMO it would have more sense to have two components `ChartShowValueCheckbox` and `ChartShowMarkers` checkbox, and in the templates put them both in the same `Section`

What do you think ?